src/libDirectoryService/FinalBlockPostProcessing.cpp

```diff
@@ -285,18 +292,16 @@ void DirectoryService::ProcessFinalBlockConsensusWhenDone() {
   }
   m_mediator.m_node->ClearUnconfirmedTxn();
 
+  AccountStore::GetInstance().InitTemp();
+  AccountStore::GetInstance().InitRevertibles();
```

Comment:
was this change intended?

Reply:
Yes. Basically we reverted the previous change. Now, Expiry window flag & state check in ProcessPowSubmission will allow submission to be accepted while node is still FinalBlockConsensus state